examples/mnist/train_test.py

```diff
@@ -21,26 +21,23 @@
 import jax
 from jax import random
 
-import numpy as onp
-
 # Parse absl flags test_srcdir and test_tmpdir.
 jax.config.parse_flags_with_absl()
 
 
 class TrainTest(absltest.TestCase):
 
-  def test_train_one_epoch(self):
-    train_ds, test_ds = train.get_datasets()
-    input_rng = onp.random.RandomState(0)
+  def test_single_train_step(self):
+    train_ds, _ = train.get_datasets()
+    batch_size = 32
     model = train.create_model(random.PRNGKey(0))
     optimizer = train.create_optimizer(model, 0.1, 0.9)
-    optimizer, train_metrics = train.train_epoch(optimizer, train_ds, 128, 0,
-                                                 input_rng)
-    self.assertLessEqual(train_metrics['loss'], 0.27)
-    self.assertGreaterEqual(train_metrics['accuracy'], 0.92)
-    loss, accuracy = train.eval_model(optimizer.target, test_ds)
```

Comment:
I think this test should still execute the `eval_model` step even thought we're only training for one step.

Reply:
(For MNIST it should be fine. For larger datasets that would be too slow, so we should at least test `eval_model` on a subset of the dataset -- say the first 32 elements)

Reply:
Yup that sounds good. Done!